docs/api/fhir-api-specification.md

```diff
@@ -1796,6 +1796,22 @@ Use `http://saraalert.org/StructureDefinition/extended-isolation` to specify a u
 }
 ```
 
+Use `http://saraalert.org/StructureDefinition/follow-up-reason` to specify a reason to follow up on the monitoree (options are: `Deceased`, `Duplicate`, `High-Risk`, `Hospitalized`, `In Need of Follow-up`, `Lost to Follow-up`, `Needs Interpretation`, `Quality Assurance`, `Refused Active Monitoring`, and `Other`).
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/follow-up-reason",
+  "valueString": "Duplicate"
+}
+```
+
+Use `http://saraalert.org/StructureDefinition/follow-up-note` to specify additional details for follow up reason on the monitoree. This requires the follow up reason to be set.
+```json
+{
+  "url": "http://saraalert.org/StructureDefinition/follow-up-note",
+  "valueString": "This is a duplicate."
+}
```

Comment:
Can you also add a follow-up reason and note to each of the patient JSON samples in the documentation for the Patient GET, POST, and PUT endpoints?